src/components/PeoplePage/PersonRow.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PersonType } from './personType';
+
+export const PersonRow = ({ person, personRows }) => (
+  <tr className="Person">
+    {personRows.map(personKey => (
```

Comment:
```suggestion
    {personСells.map(personKey => (
```
Это же не rows, ты для каждого элемента рендеришь `td`, а не `tr`